@xen-orchestra/lite/src/components/Tabs.tsx

```diff
@@ -0,0 +1,83 @@
+import Box from '@mui/material/Box'
+import React from 'react'
+import Tab from '@mui/material/Tab'
+import TabContext from '@mui/lab/TabContext'
+import TabList from '@mui/lab/TabList'
+import TabPanel from '@mui/lab/TabPanel'
+import { RouteComponentProps } from 'react-router-dom'
+import { withState } from 'reaclette'
+import { withRouter } from 'react-router'
+
+interface ParentState {}
+
+interface State {
+  pathname: string
+}
+
+interface TabType {
+  component?: React.ReactNode
+  disabled: boolean
+  label: React.ReactNode
+  pathname: string
+}
+
+interface Props extends RouteComponentProps {
```

Comment:
Why do you extend `RouteComponentProps`?

Reply:
I added a comment to make it clearer.